lib/Zef/Utils/FileSystem.pm6

```diff
@@ -54,3 +54,15 @@ sub lock-file-protect($path, &code, Bool :$shared = False) is export {
         code();
     }
 }
+
+our sub which($name) {
+    my $source-paths  := $*SPEC.path.grep(*.?chars).map(*.IO).grep(*.d);
+    my $path-guesses  := $source-paths.map({ $_.child($name) });
+    my $possibilities := $path-guesses.map: -> $path {
+        ((BEGIN $*DISTRO.is-win)
+            ?? ($path.absolute, %*ENV<PATHEXT>.split(';').map({ $path.absolute ~ $_ }).Slip)
+            !! $path.absolute).Slip
```

Comment:
Do you actually need the .Slip here? The ?? branch already results in a Slip and the !! branch results in a single value which should also be OK